app/models/invenio_rdm_record_converter.rb

```diff
@@ -138,4 +140,13 @@ def creators(creators)
       }
     end
   end
+
+  # return array of invenio formatted subjects
+  def subjects_for_scheme(terms, scheme)
+    if scheme != :tag
```

Comment:
I'm not a huge fan of this branch, but the only other options I could think of that would stay concise, which is what I liked about this solution, was to either merge in another hash (expensive, awkward) or delete the scheme key/value pair on each loop (awkward, maybe not expensive?).